.travis.yml

```diff
@@ -1,12 +0,0 @@
-language: c
```

Comment:
Added back!

> On Feb 1, 2016, at 10:57, Todd Treece <notifications@github.com> wrote:
> 
> In .travis.yml <https://github.com/adafruit/Adafruit_MQTT_Library/pull/28#discussion_r51445483>:
> 
> > @@ -1,12 +0,0 @@
> > -language: c
> no worries at all, and thanks for sharing your work! could you add the .travis.yml back to your fork when you get a chance? it would help us double check the pull req.
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/adafruit/Adafruit_MQTT_Library/pull/28/files#r51445483>.
>